uSkyBlock-Core/src/main/java/us/talabrek/ultimateskyblock/command/island/InfoCommand.java

```diff
@@ -76,7 +76,7 @@ public void run() {
                                     score.getScore(), score.getCount(),
                                     VaultHandler.getItemName(score.getBlock())));
                         }
-                        player.sendMessage(String.format("\u00a7aIsland level is %5.2f", getState().getScore()));
+                        player.sendMessage (tr(String.format("\u00a7aIsland level is %5.2f", getState().getScore())));
```

Comment:
I didn't know that "String.format" had to be removed as this was there as original code or that " %5.2f" changed to {0,number,###.##}. 
Guess that is my downfall of not knowing how to code properly :)